### DISCUSSION TITLE
When will the "glaucoma 1, open angle, E" page MONDO:0007665 be marked as obsolete?

### DISCUSSION THREAD

**Discussion by rileygr** [monarch-initiative/mondo#5925] [Q&A] [answered]
The page is currently indicated as 
**scheduled for obsoletion on or after**

- 2022-09-01

However, it does not appear in the [scheduled for obsoletion on or after](https://www.ebi.ac.uk/ols/search?q=scheduled%20for%20obsoletion%20on%20or%20after&exact=true&type=property&ontology=mondo) list

Thanks
George Riley
ClinVar Team

**Comment by nicolevasilevsky** [comment, marked as answer]
Hi George, it looks like this term is obsolete in the ontology, are you looking at this in ClinVar?

If you are looking at OLS, OLS does not have the latest Mondo release (see here: https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0007665)

If you look at either OLS4 (a newer version) or Monarch OLS, you can see the term is no longer available:
https://www.ebi.ac.uk/ols4
https://www.ebi.ac.uk/ols4/search/MONDO:0007665
